homeassistant/components/samsungtv/bridge.py

```diff
@@ -359,6 +353,22 @@ def _get_remote(self, avoid_open: bool = False) -> Remote:
                 self._remote = None
         return self._remote
 
+    def is_on(self) -> bool:
+        """Tells if the TV is on."""
+        if self._remote is not None:
+            self.close_remote()
+        try:
+            info = self.device_info()
+            if info is not None and isinstance(info["device"]["PowerState"], str):
+                return info["device"]["PowerState"] == "on"
+            return False
```

Comment:
Your current implementation assumes that if `PowerState` is absent, then the TV is offline (`return False)`.
Instead, it should fallback to the previous implemenation (`return self._get_remote() is not None`)

```suggestion
            return self._get_remote() is not None
```

Note: I have PR #67216 in progress, which also makes `is_on` abstract. As soon as that one is merged, then you will need to rebase this PR.

Reply:
#67216 is merged: please rebase